tests/testsuite/cargo_information/within_ws_and_pick_ws_package/stdout.log

```diff
@@ -0,0 +1,7 @@
+cargo-list-test-fixture
+version: 0.2.0
```

Comment:
we should append a parenthetical statement to this (in yellow?) to make it clear that this is looking at the local copy

Reply:
Nice! I will add it.

Reply:
<img width="977" alt="image" src="https://github.com/hi-rustin/cargo-information/assets/29879298/b53542a5-c1a8-4e24-a062-f2008a70251e">
I used Cyan color to print it as a note.

Reply:
But I think the note is not very clear. Do you have any suggestions?

Reply:
Updated it to:
<img width="974" alt="image" src="https://github.com/hi-rustin/cargo-information/assets/29879298/c9e7bf27-9ece-46be-b96d-52e3bd6c13be">